src/main/java/com/mabl/integration/jenkins/test/output/TestSuite.java

```diff
@@ -99,6 +115,8 @@ public String getTimestamp() {
         return this.timestamp;
     }
 
+    public List<TestCase> getTestCases() { return new ArrayList<>(testCases); }
```

Comment:
New mutable List? Do we expect consumer to be able to modify these? I could see immutable collection wrapper here, perhaps I am missing the requirement.